rpc/server.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+from xmlrpc.client import Binary
+from xmlrpc.server import SimpleXMLRPCServer
+
+path = Path(__file__).parent.parent / 'images' / 'file-protocol-1.pdf'
```

Comment:
Why do you need to open `file-protocol-1.pdf`?

Reply:
Hmm apparently you send it to client with name "logo"?

Reply:
LOL good catch!